### DISCUSSION TITLE
Can anyone recommend a vCard linter/ validator?

### DISCUSSION THREAD

**Discussion by DiagonalArg** [FossifyOrg/General-Discussion#664] [Q&A]
As discussed in this [IETF thread](https://mailarchive.ietf.org/arch/msg/vcarddav/qC8b-1J_DBM8_zNAAYc5TIG7oK8/), there is need of a vCard linter/ validator.  Can anyone recommend one so I don't have to dig too far?

**Comment by naveensingh** [comment]
I'm not aware of any user-friendly, ready-to-use vCard validators.
